### ISSUE TITLE
In Event Management -> add/edit event record -> Event details section -> Phone Number Country Code has incorrect values  

### ISSUE THREAD

**Issue by FayazMaacolony** [ijazakram/MaaColony#62]
In Event Management -> add/edit event record -> Event details section -> Phone Number Country Code has incorrect values

**Comment by Rajeswari06** [comment]
Country code is not coming from events api

**Comment by RamanaKiran** [comment]
https://test.maacolony.com/api/GCMemberApi/GetCodeList/?codeListID=27

**Comment by Rajeswari06** [comment]
Yes, the above url is giving country codes, but selected country code is not coming from the events api.

Because of this unable to show the selected country code.
